assets/js/SortFromTable.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import {store, setSelectedWfModuleAction} from "./workflow-reducer";
+import {getPageID} from "./utils";
+import WorkbenchAPI from './WorkbenchAPI'
+import {findModuleWithIdAndIdName, findParamValByIdName, getWfModuleIndexfromId} from './EditCells'
+
+var api = WorkbenchAPI();
+export function mockAPI(mock_api) {
+  api = mock_api;
+}
+
+function getNextSortDirection(current, sortType) {
```

Comment:
Too many constants here. Can you define const STRING_TYPE = 0 etc. elsewhere and use throughout? Also I think you could simplify the logic: just increment or decrement dir depending on type, then take mod 3